extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/DevServicesKafkaProcessor.java

```diff
@@ -255,18 +257,28 @@ private RunningDevService startKafka(KafkaDevServiceCfg config,
                 return new RunningDevService(Feature.KAFKA_CLIENT.getName(),
                         container.getContainerId(),
                         container::close,
-                        KAFKA_BOOTSTRAP_SERVERS, container.getBootstrapServers());
+                        getConfigs(container.getBootstrapServers(), additions));
             }
         };
 
         return maybeContainerAddress
                 .map(containerAddress -> new RunningDevService(Feature.KAFKA_CLIENT.getName(),
                         containerAddress.getId(),
                         null,
-                        KAFKA_BOOTSTRAP_SERVERS, containerAddress.getUrl()))
+                        getConfigs(containerAddress.getUrl(), additions)))
                 .orElseGet(defaultKafkaBrokerSupplier);
     }
 
+    private Map<String, String> getConfigs(String address, List<DevServiceKafkaAdditionalBuildItem> additions) {
+        HashMap<String, String> configs = new HashMap<>();
+        configs.put(KAFKA_BOOTSTRAP_SERVERS, address);
+        additions.forEach(c -> {
+            configs.put(c.getConfig(), address);
```

Comment:
The `BuildItem` is used exclusively to set the Kafka's server address as a value for other keys?

Reply:
Looking at the original issue, it seems to be `camel.component.kafka.brokers`

Reply:
yeah, we use it in  ```camel.component.kafka.brokers``` see https://github.com/apache/camel-quarkus/issues/3742

Reply:
Also you can check the detail about the motivation of these changes in https://github.com/quarkusio/quarkus/issues/25094

Reply:
I see, I am just concerned that the class name `DevServiceKafkaAdditionalBuildItem` may be too generic for this reason

Reply:
OK, it is hard to name it :) any thought ?